pubtools/_quay/operator_pusher.py

```diff
@@ -336,7 +345,7 @@ def iib_remove_operators(
                 operators, index_image
             )
         )
-        args, env_vars = cls.pubtools_iib_get_common_args(target_settings)
+        args, env_vars = cls.pubtools_iib_get_common_args(target_settings, {})
```

Comment:
Even if unused, override_settings should be added to iib_remove_operators as well.